docs/advanced/install_packages.md

```diff
@@ -15,8 +15,9 @@ A typical case is the installation of Python packages, which requires to run pip
 
 If this fails because you are trying to install an updated version of a package that already exists in CVMFS, you will need to add the `--upgrade` flag.
 
-Then, it would be necessary to add the local installation path to `PYTHONPATH`, by creating a bash startup script that configures that variable (don't forget to call this startup script in the session configuration menu):
-
+Then, it would be necessary to add the local installation path to `PYTHONPATH`. It is possible to do this, either by:
```

Comment:
I would just advertise the way of the box. The other way would need to be explained better than it is here (and than it was).

I would rephrase to:
```
After the desired Python packages have been installed on CERNBox, they need to be added to the `PYTHONPATH` when starting a SWAN session. This can be easily achieved by selecting the option "Use Python packages installed on CERNBox" under the `Software stack` section of the SWAN web form.
```
It would be even better to add an image with the option ticked after this text.